packages/babel-traverse/src/index.js

```diff
@@ -33,6 +33,7 @@ traverse.explode = visitors.explode;
 traverse.NodePath = require("./path");
 traverse.Scope    = require("./scope");
 traverse.Hub      = require("./hub");
+traverse.cache    = require("./cache");
```

Comment:
Also how do we handle environments that don't have weak maps? Does core.js
polyfill weak maps? Seems very leaky.

On Monday, 7 March 2016, Logan Smyth <notifications@github.com> wrote:

> In packages/babel-traverse/src/index.js
> <https://github.com/babel/babel/pull/3393#discussion_r55157767>:
>
> > @@ -33,6 +33,7 @@ traverse.explode = visitors.explode;
> >  traverse.NodePath = require("./path");
> >  traverse.Scope    = require("./scope");
> >  traverse.Hub      = require("./hub");
> > +traverse.cache    = require("./cache");
>
> Do we need to expose it at all, or could we call it automatically in
> addAst when a transform pass starts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/babel/babel/pull/3393/files#r55157767>.
>


-- 
Sebastian McKenzie